src/legacy/core_plugins/kibana/public/dashboard/top_nav/show_clone_modal.tsx

```diff
@@ -54,7 +54,7 @@ export function showCloneModal(
   };
   document.body.appendChild(container);
   const element = (
-    <I18nContext>
+    <I18nProvider>
```

Comment:
// brainpick

Why did you have to add `I18nProvider` in many places? Is that because of the new layer of angular added in the middle?

Reply:
Yes, I think the places I added the provider are either rendered by angular or by a direct react-dom `render` call